dumper/main.py

```diff
@@ -15,9 +15,12 @@ def get_query(query_path):
         return json.load(f)
 
 
-def init_writer(filename):
-    writer = get_csv_writer(filename)
-    writer.writerow(['domain','form_id','user_id','time_completed','time_received','delta_cs_hours'])
+def init_writer(query_choice):
+    writer = get_csv_writer(output_file)
```

Comment:
It's declared in the outer scope at the initial parsing of command-line arguments... I wasn't sure what python best-practice here is because in Java you never really have code that runs outside of a function. So I just let `es`, `index`, and `output_file` be referenced as global variables, but if it's more correct to pass them into each function as needed, I can also do that.